src/routes/controllers/users.controller.js

```diff
@@ -189,6 +190,6 @@ export async function getUsersBookmarks(req, res) {
       bookmarks: usersBookmarks,
     });
   } catch (error) {
-    return errorResponse(res, 500, error.message);
+    return serverError(res, 500, message);
```

Comment:
Since you pre-assigned statusCode to be 500 and hardcoded the message, you could simply use this function by just calling `return serverError(res)`

Reply:
Good catch... would rectify it